backend/src/controllers/classes.controller.js

```diff
@@ -0,0 +1,183 @@
+import bcrypt from "bcrypt";
+import { generateToken, generateRefreshToken, saveRefreshToken, replaceAccessToken, updateRefreshToken } from "../utils/jwt";
+import promisePool from "../../config/db";
+import setResponseJson from "../utils/responseDto";
+
+const output = {
+    classesList: async (req, res) => {
+        let userId = '';
+        if(req.user && req.user.payload) {
+            userId = req.user.payload.userId;
+        }
```

Comment:
```suggestion
        const userId = req.user?.payload?.userId ?? '';
```

https://ko.javascript.info/optional-chaining
https://ko.javascript.info/nullish-coalescing-operator